api/config/config.go

```diff
@@ -119,9 +119,8 @@ type UIConfig struct {
 	StaticPath string `validated:"required"`
 	IndexPath  string `validated:"required"`
 
-	ClockworkUIHomepage       string `json:"REACT_APP_CLOCKWORK_UI_HOMEPAGE"`
-	KubeflowUIHomepage        string `json:"REACT_APP_KUBEFLOW_UI_HOMEPAGE"`
-	CaramlAIStreamlitHomepage string `json:"REACT_APP_CARAML_AI_STREAMLIT_HOMEPAGE"`
```

Comment:
This config is removed and consolidated with the application's other configuration in the `Application.StreamlitPlaceholderPageConfig` struct.

Reply:
whats the motivation? It will be a backend config instead of ui config now?

Reply:
Nope both are UI configs. It's just that this one's served as a `env.js` file at the `/env.js` endpoint (consumed at UI startup time), while the other is served as json at the `/v2/applications` endpoint (consumed whenever the endpoint is called). I'm grouping them together so they can be configured in a single place (like we do for the Merlin/Turing apps).